src/main.rs

```diff
@@ -23,7 +23,7 @@ struct TaskItem<'r> {
 #[derive(Deserialize, Serialize)]
 #[serde(crate = "rocket::serde")]
 struct TaskId {
-    id: u8
+    id: i64
```

Comment:
This line is correct because the sqlx::query_as() method requires i64 instead of u8.